docs/source/main_classes/pipelines.rst

```diff
@@ -91,6 +92,13 @@ SummarizationPipeline
     :special-members: __call__
     :members:
 
+TableQuestionAnsweringPipeline
+=======================================================================================================================
+
+.. autoclass:: transformers.TableQuestionAnsweringPipeline
+    :special-members: __call__
+
```

Comment:
I voluntarily omitted `members` as I did not want `batch_inference` and `sequential_inference` to be documented.